README.md

```diff
@@ -49,7 +58,74 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
 > _Note_:
 > DP refers to demographic parity, EO to equalized odds, and BGL to bounded group loss. For more information on these and other terms we use in this repository please refer to the [terminology page](./TERMINOLOGY.md). To request additional algorithms or fairness definitions, please open a [new issue](https://github.com/fairlearn/fairlearn/issues).
 
-## Install fairlearn
+
+## Visualizations
+
+This repository containts a visualization dashboard that enables data scientists, auditors, business decision makers, and other stakeholders to visually observe and analyze how their model predictions are impacting different groups of data points (e.g., different ethnicities). It provides a high-level and detailed view of prediction insights for assessing which groups are negatively impacted. Using this dashboard you get insights into (un)fairness of you model and gain ideas around potential ways of mitigating the obsereved unfairness.  
```

Comment:
'containts' -> 'contains'
'to visually observe' splits an infinitive and feels tautological
'groups of data points' feels impersonal, although I can't think of anything obviously better right now

'Using this dashboard you get' is a sudden change from 'enables data scientists' to addressing the reader directly.
'you model' -> 'your models'
'gain ideas' feels a bit odd
'obsereved' -> 'observered'

Reply:
This has been edited.